app/components/chart/daily.js

```diff
@@ -117,7 +117,7 @@ var Daily = React.createClass({
     else {
       timezone = timePrefs.timezoneName || 'UTC';
     }
-    return sundial.formatInTimezone(datetime, timezone, 'dddd, MMMM Do');
+    return sundial.formatInTimezone(datetime, timezone, 'ddd, MMM D, YYYY');
```

Comment:
Do we want to keep ordinal markers after the dates? e.g., 14th instead of 14?

This is a question as much for @skrugman and @brandonarbiter 

If yes, everywhere or just daily view?